### PR TITLE
PSP-2846 - GitHub action "CI-CD PIMS Test"

### PR DESCRIPTION
**Pull request by rafaelponcedeleon**

### CODE REVIEW

File: .github/workflows/ci-cd-pims-test.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+env:
+  OPENSHIFT_SERVER: ${{ secrets.OPENSHIFT_SERVER }}
+  # service account: gitaction
+  OPENSHIFT_TOKEN: ${{ secrets.OPENSHIFT_TOKEN }}
+  OPENSHIFT_TOOLS_NAMESPACE: "3cd915-tools"
+  MS_TEAMS_WEBHOOK_BUILD_CHANNEL: ${{ secrets.MS_TEAMS_WEBHOOK_URI_BUILD_CHANNEL }}
+
+  ## variables for scripts under git\openshift\4.0\scripts\oc-*.sh
+  APP_PORT: 8080
+  DESTINATION: "dev"
```
Comment (devinleighsmith) [inline .github/workflows/ci-cd-pims-test.yml:11]: this seems incorrect. The destination for this build in jenkins is test

Comment (rafaelponcedeleon) [reply .github/workflows/ci-cd-pims-test.yml:11]: thanks for catching this. I'll update it

**Review by devinleighsmith — APPROVED** [review]
Approved, but would like @asanchezr to review this as well.

**Comment by rafaelponcedeleon** [comment]
I'll wait for @asanchezr to review before merging

**Comment by asanchezr** [comment]
Reviewing ATM

**Review by asanchezr — APPROVED** [review]
Looks good. Great work!

**Pull request merged**